platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/HttpToFile.java

```diff
@@ -0,0 +1,133 @@
+package jp.wizcorp.phonegap.plugin.WizAssets;
+import java.io.*;
+import java.net.Authenticator;
+import java.net.HttpURLConnection;
+import java.net.PasswordAuthentication;
+import java.net.URL;
+import jp.wizcorp.phonegap.plugin.WizAssets.ILogger;
+
+public final class HttpToFile {
```

Comment:
can you be more specific?  For example, downloadFile has to catch the errors to close the streams.